docs/_docs/reference/05_modifier_reference.md

```diff
@@ -79,3 +79,25 @@ Multiplies the input with the given Vector3. Useful to  control things like inpu
 Treats the input vector as a mouse position difference in pixels (e.g. from the _Mouse_ input) and converts it into a range from `(0,0)` to `(1,1)` relative to the current window size. Useful to get resolution-independent mouse differences. This modifier has no settings.
 
```

Comment:
The modifiers are sorted alphabetically according to their filename in this file, this makes it easier to see which still need documention (and to satisfy some OCD of mine). Could you move them to their sorted position, please?

Reply:
Heheh...absolutely no need to explain.  I completely understand.  I errored on the side of keeping changes together at the end so you could undo easily.  Even with Git and the fork, I still think people will be manually "undoing" things for some silly reason.  Been doing this too long, and yet, maybe not long enough?  *smirk*  Pushed an update. Alphabetized now.